third_party/github.com/bazelbuild/rules_typescript/internal/BUILD.bazel

```diff
@@ -61,6 +61,7 @@ tsc(
     data = _TSC_WRAPPED_SRCS + [
         "//internal:tsconfig.json",
         "@npm//@types/node",
+        "@npm//@angular/compiler-cli",
```

Comment:
it feels a bit undesirable to have a compile-time dependency, but it does reflect the actual types of the tsc_wrapped program where we added the layering violation

Reply:
It's only compile-them when tsc_wrapped is built as part of the `@bazel/typescript` NPM package, so it doesn't have any impact on consumers. 

The benefit is just a little type-safety here, compared to relying on`any`